insecure/wintermute/attackOrchestrator.go

```diff
@@ -34,7 +56,244 @@ func (o *Orchestrator) WithAttackNetwork(network insecure.AttackNetwork) {
 // In Corruptible Conduit Framework for BFT testing, corrupted nodes relay their outgoing events to
 // the attacker instead of dispatching them to the network.
 func (o *Orchestrator) HandleEventFromCorruptedNode(event *insecure.Event) error {
+	o.Lock()
+	defer o.Unlock()
+
+	switch protocolEvent := event.FlowProtocolEvent.(type) {
+
+	case *flow.ExecutionReceipt:
+		if err := o.handleExecutionReceiptEvent(event); err != nil {
+			return fmt.Errorf("could not handle execution receipt event: %w", err)
+		}
+	case *messages.ChunkDataRequest:
```

Comment:
can we add this comment to elaborate this case:
`// orchestrator received chunk data request from VN after VN assigned a chunk, and VN now requesting that chunk to verify it`